### DISCUSSION TITLE
头像好糊

### DISCUSSION THREAD

**Discussion by Uzawa-Reisa** [share121/inter-knot#5269] [其他]
大家也是这样吗

**Comment by Yizakl** [comment]
正常

**Comment by emmGrinch** [comment]
糊就对了，不糊你怎么知道自己用的是绳网<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
